### ISSUE TITLE
Hadamard & Total Variance

### ISSUE THREAD

**Issue by coatless** [SMAC-Group/gmwm#104]
**Feature**

We currently only have support for the allan variance and the wavelet variance. It would be nice to add in the Hadamard and Total Variance. See: [Handbook of Frequency Analysis](http://tf.nist.gov/general/pdf/2220.pdf) for more details. 

(Secondary, non-pdf source: [http://www.wriley.com/paper4ht.htm](http://www.wriley.com/paper4ht.htm) )

**Comment by coatless** [comment]
@stephaneguerrier please add paper reference

**Comment by coatless** [comment]
@avinash2692 ETA on this?

**Comment by avinash2692** [comment]
Hey James ,
Should be in by tomorrow night

Avinash

On Feb 21, 2016, at 12:59 PM, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


@avinash2692<https://github.com/avinash2692> ETA on this?

-

**Comment by coatless** [comment]
PR Tonight @avinash2692 ?

**Comment by avinash2692** [comment]
Yup , should be in tonight , wv_filters() should be on by late afternoon for pull.


On Feb 23, 2016, at 09:17, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


PR Tonight @avinash2692<https://github.com/avinash2692> ?

**Comment by coatless** [comment]
@avinash2692 

Friday sound good?

**Comment by coatless** [comment]
Missing hadamard total variance.

**Comment by avinash2692** [comment]
Working on it , will have it pushed in a couple of days

Avinash

On Feb 29, 2016, at 6:54 PM, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


Missing hadamard total variance.

-

**Comment by coatless** [comment]
Wednesday or Thursday then?

Also, look at #154 for issues that were debugged.

**Comment by avinash2692** [comment]
Yes , will be done then

Avinash

On Feb 29, 2016, at 7:52 PM, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


Wednesday or Thursday then?

Also, look at #154<https://github.com/SMAC-Group/gmwm/pull/154> for issues that were debugged.

-

**Comment by coatless** [comment]
@avinash2692 still on track for a commit tonight?

**Comment by avinash2692** [comment]
Yup, If all goes well .. which is that case now.


On Mar 3, 2016, at 14:32, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


@avinash2692<https://github.com/avinash2692> still on track for a commit tonight?

**Comment by coatless** [comment]
Excellent. Then we'll progress on to #144 next.

**Comment by coatless** [comment]
@avinash2692 Steph's explanation + the two papers is okay? Do we need to talk?

**Comment by avinash2692** [comment]
Yup , all ok ... Working on the implementation now.

Avinash

On Mar 8, 2016, at 7:35 PM, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


@avinash2692<https://github.com/avinash2692> Steph's explanation + the two papers is okay? Do we need to talk?

-
